src/GraphQL/Instrumentation/ApolloTrace.cs

```diff
@@ -3,51 +3,113 @@
 
 namespace GraphQL.Instrumentation
 {
+    /// <summary>
+    /// Contains Apollo tracing metrics.
+    /// </summary>
     public class ApolloTrace
     {
+        /// <summary>
+        /// Initializes a new instance with the specified parameters.
+        /// </summary>
+        /// <param name="start">The UTC date and time that the GraphQL document began execution.</param>
+        /// <param name="durationMs">The number of milliseconds that it took to execute the GraphQL document.</param>
         public ApolloTrace(DateTime start, double durationMs)
         {
             StartTime = start;
             EndTime = start.AddMilliseconds(durationMs);
             Duration = ConvertTime(durationMs);
         }
 
+        /// <summary>
+        /// Returns the Apollo tracing version number.
+        /// </summary>
         public int Version => 1;
 
+        /// <summary>
+        /// Returns the UTC date and time when the document began execution.
+        /// </summary>
         public DateTime StartTime { get; }
 
+        /// <summary>
+        /// Returns the UTC date and time when the document completed execution.
+        /// </summary>
         public DateTime EndTime { get; }
 
+        /// <summary>
+        /// Returns the duration of the document's execution, in nanoseconds.
+        /// </summary>
         public long Duration { get; }
 
+        /// <summary>
+        /// Returns the parsing metrics.
+        /// </summary>
         public OperationTrace Parsing { get; } = new OperationTrace();
 
+        /// <summary>
+        /// Returns the validation metrics.
+        /// </summary>
         public OperationTrace Validation { get; } = new OperationTrace();
 
+        /// <summary>
+        /// Returns the execution metrics.
+        /// </summary>
         public ExecutionTrace Execution { get; } = new ExecutionTrace();
 
+        /// <summary>
+        /// Converts a quantity of milliseconds to nanoseconds.
+        /// </summary>
         public static long ConvertTime(double ms) => (long)(ms * 1000 * 1000);
```

Comment:
This method should not be public. May be done in separate PR targeted to `develop`.

Reply:
Done in #2083